common/script-variable-generator/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
common/..  is the project structure I originally had. In https://github.com/spring-cloud/spring-cloud-stream-modules/pull/12  I went with spring-cloud-stream-module-common-configuration and spring-cloud-stream-module-test-support.  I'm okay with standardizing on this, but a lot of pom changes required. Please confirm.